docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1391,25 +1403,39 @@ Summary of Iron Intervention Algorithm
   mother_iron_group   = ('baseline' if mother_fortified_baseline_i else
                          'intervention_not_baseline' if mother_fortified_i else
                          'uncovered')
-  bw_shift_i = bw_response * iron_concentration * daily_flour_i
-  birth_weight_i      = birthweight_gbd_i + (
-                        -baseline_coverage(t) + mother_fortified_i
-                        ) * bw_shift_i
-
-  # At each simulation time step (including when simulant is born)
-  child_fortified_i(t)  = time_covered_i <= t
-                        = p_i < coverage(t)
-    # If child is covered in baseline, then time_covered_i = float('-inf'),
-    # and hence hb_lag_fraction(t-time_covered_i) = 1, i.e. the child receives
-    # the full age-dependent effect in this case since the time lag has passed.
-  hb_i(t)               = hb_gbd(age_i(t)) + (
-                          -baseline_coverage(t) + child_fortified_i(t) * hb_lag_fraction(t-time_covered_i)
-                          ) * hb_shift *  hb_age_fraction(age_i(t))
+    # Effect on birthweight
+  daily_flour_i       = average daily flour consumption Y of simulant's mother
+                      = (flour_consumption_dist.rvs() if mother_fortified_i else
```

Comment:
```suggestion
                      = (flour_consumption_dist.rvs() if mother_fortified_i == True else
```

Reply:
Well, perhaps `mother_is_fortified` would be a better name for a boolean so that `if mother_is_fortified` reads even more naturally, but I was trying to save space...

https://softwareengineering.stackexchange.com/questions/12807/make-a-big-deal-out-of-true

Reply:
Got it! This is just my lack of familiarity, so thanks for orienting me :)